integration/vm_cache/vm_cache_test.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This test will perform several tests to validate kata containers with
+# vm cache.
+
+set -e
+
+cidir=$(dirname "$0")
+
+source "${cidir}/../../metrics/lib/common.bash"
+
+# Environment variables
+IMAGE="${IMAGE:-busybox}"
+PAYLOAD_ARGS="${PAYLOAD_ARGS:-tail -f /dev/null}"
+kata_runtime_bin=$(command -v kata-runtime)
+
+if [ -z $INITRD_PATH ]; then
+	echo "Skipping vm templating test as initrd is not set"
+	exit 0
+fi
+
+enable_vm_cache_config() {
+	echo "enable vm cache config"
+	sudo sed -i 's/#vm_cache_number = 0/vm_cache_number = 1/g' "${RUNTIME_CONFIG_PATH}"
+}
+
+disable_vm_cache_config() {
+	echo "disable vm cache config"
+	sudo sed -i 's/vm_cache_number = 1/#vm_cache_number = 0/g' "${RUNTIME_CONFIG_PATH}"
+}
+
+init_vm_template() {
+	echo "init vm template"
+	sudo -E PATH=$PATH "$kata_runtime_bin" factory init
```

Comment:
Is running factory init twice intentional? If so, do we not need to check the return code of the first one?

Reply:
@ganeshmaharaj not worries, thanks for the feedback